TARGET_STM32WL/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@ add_subdirectory(TARGET_LORA_E5_MINI EXCLUDE_FROM_ALL)
 add_subdirectory(TARGET_LORA_E5_DEV_BOARD EXCLUDE_FROM_ALL)
 add_subdirectory(TARGET_LORA_E5_TINY EXCLUDE_FROM_ALL)
 add_subdirectory(TARGET_RAK3172 EXCLUDE_FROM_ALL)
-add_subdirectory(TARGET_RAK3172_BREAKOUT EXCLUDE_FROM_ALL)
+add_subdirectory(TARGET_RAK3272_BREAKOUT EXCLUDE_FROM_ALL)
```

Comment:
Should be TARGET_RAK3272_SIP_BREAKOUT?
TARGET_RAK3172_BREAKOUT is wrongly deleted?
TARGET_RAK3172SIP ?

Reply:
> Should be TARGET_RAK3272_SIP_BREAKOUT?

yeah could be, gave me headache with commercial names because there is SIP and SIPLP so I left generic one, in many cases should be adapted in local `custom_targets.json` so I left this one

>TARGET_RAK3172_BREAKOUT is wrongly deleted?

fixed

> TARGET_RAK3172SIP ?

fixed